packages/shared/src/components/notifications/FirstNotification.tsx

```diff
@@ -20,6 +20,7 @@ function FirstNotification(): ReactElement {
       isUnread={isUnread}
       type={NotificationType.System}
       icon={NotificationIcon.Bell}
+      targetUrl="#"
```

Comment:
I dont think you should do that

Reply:
There is nothing to do here in the first notification. Not sure if there should be any action here.

Reply:
We still don't have the rebrandly link from the product, let's add it in the intermediate branch so we can close out this PR.

Reply:
Yeah, ok for me 👍

Reply:
Thank you for the review! 🚢